src/mrchat-private.h

```diff
@@ -61,7 +61,8 @@ int             mrchat_are_all_members_verified__ (mrchat_t*);
 
 
 #define         MR_CHAT_PREFIX              "Chat:"      /* you MUST NOT modify this or the following strings */
-#define         MR_CHATS_FOLDER             "Chats"      /* if we want to support Gma'l-labels - "Chats" is a reserved word for Gma'l */
+#define         MR_CHATS_FOLDER             "DeltaChats" /* Folder where the Chats will be moved to */
```

Comment:
i think the name should be `DeltaChat` (without trailing `s`) and not `DeltaChats`